workshops/models.py

```diff
@@ -1424,3 +1426,197 @@ def long_status(self):
+    )
+    other_field = models.CharField(
+        max_length=STR_LONG,
+        verbose_name=' ',
+        null=False, blank=True, default='',
+    )
+    return field, other_field
+
+
+@reversion.register
+class TrainingRequest(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    personal = models.CharField(
+        max_length=STR_LONG,
+        verbose_name='Personal name',
+        blank=False,
+    )
+    family = models.CharField(
+        max_length=STR_LONG,
+        verbose_name='Family name',
+        blank=False,
+    )
+
+    email = models.EmailField(
+        verbose_name='Email address',
+        blank=False,
+    )
+
+    # TODO: DRY (ProfileUpdateRequest)
+    OCCUPATION_CHOICES = (
+        ('undisclosed', 'Prefer not to say'),
+        ('undergrad', 'Undergraduate student'),
+        ('grad', 'Graduate student'),
+        ('postdoc', 'Post-doctoral researcher'),
+        ('faculty', 'Faculty'),
+        ('research', 'Research staff (including research programmer)'),
+        ('support', 'Support staff (including technical support)'),
+        ('librarian', 'Librarian/archivist'),
+        ('commerce', 'Commercial software developer '),
+        ('', 'Other (enter below)'),
+    )
+    occupation = models.CharField(
+        max_length=STR_MED,
+        choices=OCCUPATION_CHOICES,
+        verbose_name='What is your current occupation/career stage?',
+        help_text='Please choose the one that best describes you.',
+        null=False, blank=True, default='undisclosed',
+    )
+    occupation_other = models.CharField(
+        max_length=STR_LONG,
+        verbose_name='Other occupation/career stage',
+        blank=True, default='',
+    )
+
+    affiliation = models.CharField(
+        max_length=STR_LONG,
+        verbose_name='Institutional Affiliation',
+        null=False, blank=False,
+    )
+
+    location = models.CharField(
+        max_length=STR_LONG,
+        verbose_name='Location',
+        help_text='please give city, province or state',
+        blank=False,
+    )
+    country = CountryField()
+
+    domains = models.ManyToManyField(
+        'KnowledgeDomain',
+        verbose_name='Areas of expertise',
+        help_text='Please check all that apply.',
+        limit_choices_to=~Q(name__startswith='Don\'t know yet'),
+        blank=True,
+    )
+    domains_other = models.CharField(
+        max_length=255,
+        verbose_name='Other areas of expertise',
+        blank=True, default='',
+    )
+
+    GENDER_CHOICES = (
+        (Person.UNDISCLOSED, 'Prefer not to say'),
+        (Person.FEMALE, 'Female'),
+        (Person.MALE, 'Male'),
+        (Person.OTHER, 'Other (enter below)'),
+    )
+    gender = models.CharField(
+        max_length=1,
+        choices=GENDER_CHOICES,
+        null=False, blank=False, default=Person.UNDISCLOSED,
+    )
+    gender_other = models.CharField(
+        max_length=STR_LONG,
+        verbose_name=' ',
```

Comment:
It's left empty, because it looks good in the form. See the example of `gender`:
![image](https://cloud.githubusercontent.com/assets/1645996/15635642/ff55d5b0-25e4-11e6-9d0c-98f68547d2b8.png)

In the case of areas of expertise, there is no "Other (enter below)" option, so it could be not clear what the text field is for if we left the label empty.